lib/quantum/normalizer.ex

```diff
@@ -28,6 +32,12 @@ defmodule Quantum.Normalizer do
   defp normalize_job({job_name, %Quantum.Job{} = job}) do
     # Sets defauts for job if necessary
     job_name |> job_opts([]) |> Map.merge(%{job | name: job_name})
```

Comment:
Just want to mention, that this overrides the name property regardless of the value that was or was not specified by the user, which means it is not 'defaulting', but rather 'overriding'.